cores/esp32/esp32-hal-tinyusb.c

```diff
@@ -61,7 +61,12 @@ typedef struct {
 
 static bool usb_otg_deinit(void * busptr) {
     // Once USB OTG is initialized, its GPIOs are assigned and it shall never be deinited
+    // except when S3 swithicng usb from cdc to jtag while resetting to bootrom
+#if CONFIG_IDF_TARGET_ESP32S3
+    return true;
```

Comment:
A better fix could be using if check `usb_persist_mode==RESTART_BOOTLOADER` in the function, though we will need to move the `static restart_type_t usb_persist_mode` before usb_otg_deinit(). I am sure you guys could figure a better way. This PR is just a quick hack/suggestion to prove the point